packages/variant-theme/src/index.ts

```diff
@@ -1 +1 @@
-export {};
+export { getVariantTheme } from './ThemeGenerator';
```

Comment:
this entails non trivial amount of logic, I'd suggest adding tests for this otherwise how can you guarantee that his will work ?

Reply:
Tests are planned! Do you think they need to be in the first merge or can they be included in a follow-up?

Reply:
it's up to the owners 🙌, I personally cannot verify if the logic works as intended so I'd require them to be added, but feel free to continue without for now.